Xamarin.Forms.Core/Picker.cs

```diff
@@ -290,5 +306,10 @@ void ITextElement.OnCharacterSpacingPropertyChanged(double oldValue, double newV
 			InvalidateMeasure();
 		}
 
+		public void OnHorizontalTextAlignmentPropertyChanged(TextAlignment oldValue, TextAlignment newValue)
```

Comment:
From the instructions in the [`interface ITextAlignmentElement`](https://github.com/xamarin/Xamarin.Forms/blob/6ee4ec697454c9a8268759dc61abf0bc5c8cc829/Xamarin.Forms.Core/ITextAlignmentElement.cs) you should implement this method explicitly.

Reply:
Thanks. Done.